Update create.md to reflect new page limit (#6950)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/in-app_messages/drag_and_drop/create.md

3. Name the page something meaningful. This will help you when connecting pages together.

{% alert tip %}

> There is a limit of 10 pages per in-app message.

Reviewer: ```suggestion
You can add up to 10 pages per in-app message.
```
